### PR TITLE
docs/enterprise: update thales install docs

### PR DESCRIPTION
**Pull request by ameets**
- update machine key for tct2 command
	- include instructions for loading machine image and user files
	  from HSM front panel

### CODE REVIEW

**Comment by ameets** [comment]
`tct2 --print-sigs xprvseemodule.sar` prints the key used to sign the file

File: docs/enterprise/thales-install-guide.md [hunk 1]
```diff
@@ -38,15 +38,24 @@ You must also perform the following steps:
...
+   - `tct2 --sign-and-pack -k xprvseemoduledevusk --machine-key=02cf0cd76a8e726ffba2e4a1d2648b447d505aaa -o userdata.sar -i userdata.bin`
    
    The hex value here is the hash of the key that Chain used to sign
    the SEE machine. **Important**: this hash is of a _development key_ and
    should not be trusted for production.
    
--  On the client host, use the “nfast” utilities from Thales to
-   install the signed firmware and userdata files. This command must be run on each module. The `-m` flag specifies the module.
-   - `loadsee-setup -s -m 1 -M xprvseemodule.sar`
+-  On the client host, copy the signed firmware and userdata files to the custom-seemachines directory.
```
Comment (ryandotsmith) [inline docs/enterprise/thales-install-guide.md]: Perhaps mention that this should be the RFS too?

File: docs/enterprise/thales-install-guide.md [hunk 2]
```diff
@@ -38,15 +38,24 @@ You must also perform the following steps:
...
    The hex value here is the hash of the key that Chain used to sign
    the SEE machine. **Important**: this hash is of a _development key_ and
    should not be trusted for production.
    
--  On the client host, use the “nfast” utilities from Thales to
-   install the signed firmware and userdata files. This command must be run on each module. The `-m` flag specifies the module.
-   - `loadsee-setup -s -m 1 -M xprvseemodule.sar`
+-  On the client host, copy the signed firmware and userdata files to the custom-seemachines directory. 
+   This command must be run on each module.
+   - `cp xprvseemodule.sar /opt/nfast/custom-seemachines/a`
```
Comment (ryandotsmith) [inline docs/enterprise/thales-install-guide.md]: Maybe add a note why we name things `a` and `b`?

File: docs/enterprise/thales-install-guide.md (hunk 1 above)
Comment (ameets) [inline docs/enterprise/thales-install-guide.md]: Something like "On the client host (RFS server)..." ? Or replace "client host" with "RFS server" to be more specific?

**Pull request merged**